spk/sickchill/Makefile

```diff
@@ -1,23 +1,23 @@
 SPK_NAME = sickchill
 SPK_VERS = 20210729
-SPK_REV = 2
+SPK_REV = 3
 SPK_ICON = src/sickchill.png
 
-DEPENDS = cross/$(SPK_NAME)
-SPK_DEPENDS = "python38:git"
-PIP=pip3
+BUILD_DEPENDS = cross/python38 cross/poetry cross/lxml
+# cross/cffi cross/setuptools cross/cryptography
+DEPENDS = cross/PyNaCl cross/$(SPK_NAME)
+SPK_DEPENDS = "python38"
 
-WHEELS = src/requirements.txt
+WHEELS = src/requirements-cross.txt src/requirements.txt
 
 MAINTAINER = miigotu
 DESCRIPTION = Automatic Video Library Manager for TV Shows. It watches for new episodes of your favorite shows, and when they are posted it does its magic.
-DESCRIPTION_SPN = Gestor automático para bibliotecas de series. Busca nuevos episodios de tus series favoritas, y cuando son publicados hace su magia.
 STARTABLE = yes
 DISPLAY_NAME = SickChill
-CHANGELOG = "Initial SickChill env fix and move to Python 3.8"
+CHANGELOG = "Initial SickChill env fix and move to Python 3.8 without git"
```

Comment:
```suggestion
CHANGELOG = 1. SickChill environment fix<br>2. Move to Python 3.8 without git<br>3. Added DSM7 Support
```

Reply:
Not important but why not?

Reply:
Sure some house keeping and details of where we are